app/pb_kits/playbook/pb_typeahead/components/MultiValue.jsx

```diff
@@ -0,0 +1,45 @@
+/* @flow */
+
+import React from 'react'
+import { components } from 'react-select'
+
+import { FormPill } from '../../'
+
+type Props = {
+  data: object,
+  multiValueTemplate: any,
+  removeProps: any,
+}
+
+const MultiValue = (props: Props) => {
+  const {
+    data,
+    removeProps,
+  } = props
+
+  const handleOnClick = removeProps.onClick
+
+  return (
+    <components.MultiValueContainer
+        className="text_input_multivalue_container"
+        {...props}
+    >
+      <If condition={data.avatarUrl}>
```

Comment:
Looks like the Typeahead component is not supposed to be user-specific, in which case I'd make `avatarUrl` something a little more generic. I suppose `avatarURL` could be the URL of a product image in the case of selecting products.

Perhaps ~`data.avatarUrl`~ would better convey the intention as `data.imageUrl`?

Reply:
That could work!

Reply:
@drborges done!